### PR TITLE
fix the checking bug in usageUtils

### PR DESCRIPTION
**Pull request by liliuleo93**
the checking will be useless if empty value is `0.0`, please let me know if we should check more

### CODE REVIEW

File: processing/src/main/java/org/apache/druid/query/UsageUtils.java [hunk 1]
```diff
@@ -110,11 +110,27 @@ public static void incrementAuSignals(AtomicLong numAuSignals, List<ColumnValueS
...
         continue;
       }
       columnInvolved++;
     }
     numAuSignals.addAndGet(columnInvolved);
   }
+
+  private static boolean isEmpty(Object value)
+  {
+    if (value == null || "".equals(value) || "null".equals(value)) {
```
Comment (joykent99) [inline processing/src/main/java/org/apache/druid/query/UsageUtils.java]: I am curious in which case a null value is stored as string "null"?

File: processing/src/main/java/org/apache/druid/query/UsageUtils.java [hunk 2]
```diff
@@ -110,11 +110,27 @@ public static void incrementAuSignals(AtomicLong numAuSignals, List<ColumnValueS
...
+      return true;
+    }
+
+    if (value instanceof Double
+        || value instanceof Long
+        || value instanceof Integer
+        || value instanceof Short
+        || value instanceof Float) {
+      return (double) value == 0;
```
Comment (joykent99) [inline processing/src/main/java/org/apache/druid/query/UsageUtils.java]: Change it to

```
return ((double) value) == 0;
```

File: processing/src/main/java/org/apache/druid/query/UsageUtils.java (hunk 1 above)
Comment (liliuleo93) [reply processing/src/main/java/org/apache/druid/query/UsageUtils.java]: No idea, I am not sure what will be the default value

File: processing/src/main/java/org/apache/druid/query/UsageUtils.java (hunk 2 above)
Comment (liliuleo93) [reply processing/src/main/java/org/apache/druid/query/UsageUtils.java]: sure

File: processing/src/main/java/org/apache/druid/query/UsageUtils.java (hunk 1 above)
Comment (joykent99) [reply processing/src/main/java/org/apache/druid/query/UsageUtils.java]: A column can be String, Double, Float, Integer, Long, as far as I know.

If you don't know if "null" represents null value, let us remove it.

**Pull request merged**